File: Python/lista3 - listas/exercicio_8.py
a = []
for i in range(0, 10):
    a.append(int(input('Informe um numero: ')))

sum_of_square = 0
for number in a:
    sum_of_square += (number * number)

print('A soma dos quadrados dos numeros lidos eh %d' % sum_of_square)

File: C/lista3 - vetores/exercicio_8.c
#include <conio.h>
#include <stdio.h>

int main() {
	int i, numbers[10], sumOfSquares = 0;
	
	for (i= 0; i< 10; i++) {
		printf("Informe um numero: ");
		scanf("%d", &numbers[i]);
		
		sumOfSquares += (numbers[i] * numbers[i]);
	}
	
	printf("A soma dos quadrados dos numeros lidos: %d", sumOfSquares);
	
	getch();
	return 0;
}